Dockerfile

```
@@ -13,18 +14,24 @@ ADD ./scripts/check-rpc.sh /usr/local/bin/check-rpc.sh
 ADD ./scripts/check-sync.sh /usr/local/bin/check-sync.sh
 RUN chmod a+x /usr/local/bin/*.sh
 
-#Change default monero UID and GID, disable user's home and shell
-RUN sed -i "s|monero:x:1000:1000:Linux User,,,:/home/monero:/bin/bash|monero:x:302340:302340:Monero:/dev/null:/sbin/nologin|" /etc/passwd
-RUN sed -i "s/^\(monero:x\):1000:$/\1:302340:/" /etc/group
+#Modify gid & uid of monero user & group:
+RUN sed -i "s|^monero:x:1000:1000:Linux User,,,:/home/monero:/bin/bash|monero:x:30234:302340:Monero:/dev/null:/sbin/nologin|" /etc/passwd
+RUN sed -i "s|^\(monero:x\):1000:$|\1:302340:|" /etc/group
+RUN sed -i "s|^\(monerowallet:x:30235:302340:\)Linux User,,,\(:/dev/null:/sbin/nologin\)|\1Monero Wallet RPC User\2|" /etc/passwd
+RUN adduser -h /dev/null -s /sbin/nologin -D -H -u 30235 -G monero monerowallet
 
 # # Add config file for monerod
-COPY ./assets/monero.conf.template /root/
+COPY ./assets/*.conf.template /root/
+
+# # Expose p2p, unrestricted RPC, ZMQ, ZMQ-PUB, and restricted RPC ports
+#EXPOSE 18080/tcp
+#EXPOSE 18081/tcp
+#EXPOSE 18082/tcp
+#EXPOSE 18083/tcp
+##EXPOSE 18089/tcp
```

```suggestion
#EXPOSE 18089/tcp
```

perhaps an extra #? Unsure if intentional

---

I did intentionally have two `#`s.  We're no longer exposing the unrestricted rpc  (on 18089) to other services' containers.
But I was mistaken to comment the rest of these EXPOSE lines in the Dokerfile -- they're required.
Fixed by [d046107](https://github.com/kn0wmad/monerod-startos/pull/32/commits/d0461077194a763bed004024b9fd97ca799864df)